tests/Api/Request/ParametersBuilderTest.php

```diff
@@ -83,7 +83,35 @@ public function setSearchParamsProvider()
                         0 => 'variation_id'
                     ],
                     'attrib' => [
-                        'color' => ['red', 'blue']
+                        'color' => ['red', 'blue'],
+                        'cat' => ['Category 2']
```

Comment:
Why was this added? The other parameters make sense, but why do we need a category now?

Reply:
We have commented out the continue in the [src/Api/Request/ParametersBuilder](https://github.com/findologic/plugin-plentymarkets-ceres-api/pull/303/files#diff-e93f58cf887da50267647fd05d8598121210af7bb2924259e16341590c69d567)::87. Now we are setting cat attribute to the request, so I had to add it here too.